buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -393,80 +325,96 @@
     }
   ],
   "variables": {
-    "system.debug": {
-      "value": "false",
+    "CloudDropAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "OfficialBuildId": {
+      "value": "$(Build.BuildNumber)",
       "allowOverride": true
     },
-    "PB_GitDirectory": {
-      "value": "/root/corefx"
+    "PB_BuildArguments": {
+      "value": "-buildArch=x64 -Release -stripSymbols",
+      "allowOverride": true
+    },
+    "PB_BuildTestsArguments": {
+      "value": "-buildArch=x64 -Release -SkipTests -Outerloop -- /p:ArchiveTests=true /p:EnableDumpling=true",
+      "allowOverride": true
+    },
+    "PB_CleanAgent": {
+      "value": "true"
+    },
+    "PB_CloudDropAccountName": {
+      "value": "dotnetbuildoutput"
+    },
+    "PB_CloudResultsAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_CloudResultsAccountName": {
+      "value": "dotnetjobresult"
+    },
+    "PB_CreateHelixArguments": {
+      "value": "\"/p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:TestProduct=corefx /p:TimeoutInSeconds=1200 /p:TargetOS=Linux",
+      "allowOverride": true
+    },
+    "PB_DockerCommonRunArgs": {
+      "value": "--name $(PB_DockerContainerName) -v \"$(PB_GitDirectory):$(PB_DockerVolumeName)\" -w=\"$(PB_DockerVolumeName)\" $(PB_DockerImageName)"
```

Comment:
I didn't test that explicitly, run times for the Linux builds appear to be appx on par - https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=855235

I neglected to update Linux-Crossbuild with this change, I'm rectifying (validating) that change right now.